librz/core/cbin.c

```diff
@@ -2095,10 +2095,10 @@ static int bin_imports(RzCore *r, PJ *pj, int mode, int va, const char *name) {
 		rz_table_set_columnsf(table, "nXssss", "nth", "vaddr", "bind", "type", "lib", "name");
 	}
 	rz_list_foreach (imports, iter, import) {
-		if (name && strcmp(import->name, name)) {
+		if ((name && strcmp(import->name, name)) || !import->name) {
```

Comment:
if there is the chance that `import->name` is NULL, shouldn't we check before actually using it in the `strcmp`?

Reply:
oh, you are right.